browser/src/Services/Explorer/ExplorerSelectors.ts

```diff
@@ -50,6 +50,20 @@ export const isPathExpanded = (state: IExplorerState, pathToCheck: string): bool
     return !!state.expandedFolders[pathToCheck]
 }
 
+/**
```

Comment:
@feltech a version of this same functionality already exists in [`ExplorerStore.ts`] (https://github.com/Akin909/oni/blob/c8a6051e390449873a1ed80e385643ad11adfe8b/browser/src/Services/Explorer/ExplorerStore.ts#L392) maybe you could import that and use it instead